lib/solReleases.js

```diff
@@ -0,0 +1,40 @@
+const map = require('../lib/mapping.json');
+const rp = require('request-promise');
+
+function getCompilerVersion(parsedData) {
+    return new Promise(async function(resolve, reject){
+        if(parsedData[0].hasOwnProperty('start_version') == false) {
+            reject(new Error('No Pragma specified !!!'))
```

Comment:
All Error messages follow the first case capital convention. Please change `specified` to `Specified`